thundra/integrations/botocore.py

```diff
@@ -321,13 +356,13 @@ def getRequestType(self, string):
         return Constants.AWS_SERVICE_REQUEST
 
     def inject_span_info(self, scope, wrapped, instance, args, kwargs, response,
-                 exception):
+                         exception):
 
         operation_name, request_data = args
         self.lambdaFunction = request_data.get('FunctionName', '')
         scope.span.domain_name = Constants.DomainNames['API']
         scope.span.class_name = Constants.ClassNames['LAMBDA']
-        scope.span.operation_name = self.getRequestType(operation_name)
+        scope.span.operation_name = self.lambdaFunction
```

Comment:
what if there is no FunctionName in the request? For example, when the request is for list functions